assets/js/components/wp-dashboard/WPDashboardPopularPages.js

```diff
@@ -39,45 +44,66 @@ import { numFmt } from '../../util';
 const { useSelect } = Data;
 
 const WPDashboardPopularPages = ( { WidgetReportZero, WidgetReportError } ) => {
-	const dateRangeDates = useSelect( ( select ) =>
-		select( CORE_USER ).getDateRangeDates( {
+	const { report, titles, loading, error } = useSelect( ( select ) => {
+		const dateRangeDates = select( CORE_USER ).getDateRangeDates( {
 			compare: true,
 			offsetDays: DATE_RANGE_OFFSET,
-		} )
-	);
+		} );
+		const reportArgs = {
+			...dateRangeDates,
+			metrics: [
+				{
+					expression: 'ga:pageviews',
+					alias: 'Pageviews',
+				},
+			],
+			dimensions: [ 'ga:pagePath' ],
+			orderby: [
+				{
+					fieldName: 'ga:pageviews',
+					sortOrder: 'DESCENDING',
+				},
+			],
+			limit: 5,
+		};
+		const data = {
+			report: select( MODULES_ANALYTICS ).getReport( reportArgs ),
+			error: select( MODULES_ANALYTICS ).getErrorForSelector(
+				'getReport',
+				[ reportArgs ]
+			),
+			loading: true,
+		};
+		const reportLoaded = select(
+			MODULES_ANALYTICS
+		).hasFinishedResolution( 'getReport', [ reportArgs ] );
 
-	const reportArgs = {
-		...dateRangeDates,
-		metrics: [
-			{
-				expression: 'ga:pageviews',
-				alias: 'Pageviews',
-			},
-		],
-		dimensions: [ 'ga:pageTitle', 'ga:pagePath' ],
-		orderby: [
-			{
-				fieldName: 'ga:pageviews',
-				sortOrder: 'DESCENDING',
-			},
-		],
-		limit: 5,
-	};
+		let hasLoadedPageTitles = true;
```

Comment:
This should be `false` by default, shouldn't it?

Reply:
Updated in https://github.com/google/site-kit-wp/pull/3958/commits/3cddcca994d8eb6a2e939ac3aca4998081ba16f2

Though it could go either way, since on an initial load, there is no report, and with no report there are no pageTitles to load. Probably makes more sense for `false` though.